.github/workflows/deploy.yml

```diff
@@ -42,14 +42,18 @@ jobs:
         run: |
           invoke virtualenv
           invoke run -t update-downloads
-          invoke run -t update-docs
+
```

Comment:
This looks wrong, or did you somehow manage to make this happen automatically with `doc`, which I don't think should be done though, as it requires making checkouts and web queries that generally take too long for mere mortals to be doing that.

Reply:
`invoke run -t update-docs` is disabled because the layout of the web page is broken.

Reply:
That shouldn't be the case, normally it should not change anything. On `main` it didn't change anything at all the last time I ran it, before merging the branch. It is only supposed to update changelog pages and manuals, and credits file from the repo, applying a few changes to them.